stdlib/time/src/main/java/org/ballerinalang/stdlib/time/nativeimpl/Parse.java

```diff
@@ -46,7 +48,10 @@
         functionName = "parse",
         args = {@Argument(name = "timestamp", type = TypeKind.STRING),
                 @Argument(name = "format", type = TypeKind.UNION)},
-        returnType = {@ReturnType(type = TypeKind.RECORD, structType = "Time", structPackage = "ballerina/time")},
+        returnType = {
+                @ReturnType(type = TypeKind.RECORD, structType = "Time", structPackage = "ballerina/time"),
+                @ReturnType(type = TypeKind.RECORD, structType = "error", structPackage = BALLERINA_BUILTIN_PKG)
```

Comment:
Is Type Kind should be  TypeKind.ERROR? Same with other native functions as well.